documentation/oracle/dvm_interfaces.md

```diff
@@ -134,17 +134,19 @@ price request for that `identifier` and `time` is returned from `getPendingReque
 `Commit`.
 
 It also takes the `hash` that the voter wishes to commit to. The `hash` is created by computing
-`keccak256(price, salt)`, where price is the `int256` price value that they wish to submit and `salt` is a random
-`int256` value. The voter must remember the `price` and `salt` that they submitted so they can reveal their commit
-later - otherwise, the commit cannot be revealed and the vote won't be counted.
+`keccak256(price, salt, time, address, roundId, identifier)`, where price is the `int256` price value that they wish to submit, `time` is the unix timestamp of the request being voted on, `address` is the voter's address (technically, the same one that will reveal the vote), `roundId` is the current voting round, `identifier` is the pricefeed identifier relevant to the price request, and `salt` is a random `int256` value. The voter must remember the inputs to the vote `hash` that they submitted so they can reveal their commit later - otherwise, the commit cannot be revealed and the vote won't be counted.
```

Comment:
@rcai1 I changed this description to `commitVote` to reflect that  the vote hash now requires additional information.

Reply:
Makes sense!